### PR TITLE
Replace FunctionClarity with VMClarity

### PR DESCRIPTION
**Pull request by edwarnicke**
Signed-off-by: Ed Warnicke <hagbard@gmail.com>

### CODE REVIEW

File: components/Layout.tsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import Navbar from "./globals/Navbar";
 import Footer from "./globals/Footer";
 import { NextSeo } from "next-seo";
 
-export const siteTitle = "APIClarity";
+export const siteTitle = "OpenClarity";
```
Comment (edwarnicke) [inline components/Layout.tsx:9]: This changes the title of the page in the browser bar from API Clarity to OpenClarity:

image

Comment (edwarnicke) [reply components/Layout.tsx:9]: Current (APIClarity) on the left, new (OpenClarity) on the right.

File: data/whatIsData.tsx [hunk 2]
```diff
@@ -2,24 +2,26 @@ import { ImgPaths } from "../types/enums";
 
 export const whatIsData = [
   {
-    title: "FunctionClarity",
+    title: "VMClarity",
     description:
-      "FunctionClarity is an open source tool that provides a framework for signing and validating serverless cloud function artifacts (images or code) utilizing various signing techniques, including keyless.",
+      "VMClarity is an open source tool for agentless detection and management of Virtual Machine Software Bill Of Materials (SBOM) and security threats such as vulnerabilities, exploits, malware, rootkits, misconfigurations and leaked secrets.",
     keyCapabilities: [
```
Comment (edwarnicke) [inline data/whatIsData.tsx:8]: Copy for the VMClarity section on the main page

File: data/whatIsData.tsx [hunk 3]
```diff
@@ -2,24 +2,26 @@ import { ImgPaths } from "../types/enums";
...
-      "Keyless and key-pair support",
-      "Multiple validation feedback support (detect, block, and notification)",
+      "SBOM analysis",
+      "Package and OS vulnerability detection",
+      "Exploit detection",
+      "Leaked secret detection",
+      "Malware detection",
+      "Misconfiguration detection",
+      "Rootkit detection",
     ],
```
Comment (edwarnicke) [inline data/whatIsData.tsx:16]: Copy for the 'Key Features' for VMClarity.

File: pages/index.tsx [hunk 4]
```diff
@@ -103,8 +103,7 @@ const HeroBanner = () => {
               <h1>What is OpenClarity?</h1>
               <h2>
                 OpenClarity is a suite of open source tools for cloud native
-                security and observability—APIClarity, KubeClarity, and
-                FunctionClarity.
+                security and observability—APIClarity, KubeClarity and VMClarity.
```
Comment (edwarnicke) [inline pages/index.tsx:106]: Copy for 'What is OpenClarity?'

**Comment by KimMcMahon** [comment]
LGTM #shipit

File: components/Layout.tsx (hunk 1 above)
Comment (edwarnicke) [reply components/Layout.tsx:9]: Replaced with new 'eye' favicon:

image

image

File: components/Layout.tsx [hunk 5]
```diff
@@ -36,7 +36,7 @@ const Layout = ({
 				{/*<!--Tracking Code End of HubSpot Embed Code -->*/}
 			</Head>
 			<NextSeo
-				title={`APIClarity | ${pageTitle}`}
+				title={`OpenClarity | ${pageTitle}`}
 				description="Open source for API traffic visibility in K8s clusters"
```
Comment (pkalapat) [inline components/Layout.tsx:40]: Might want to update this description

File: components/Layout.tsx [hunk 6]
```diff
@@ -36,7 +36,7 @@ const Layout = ({
...
 				canonical={`https://apiclarity.io${pagePath || ""}`}
```
Comment (pkalapat) [inline components/Layout.tsx:41]: Might want to remove APIclarity reference here.

**Review by amccormi — APPROVED** [review]
This looks good to me

**Review by jesqi — APPROVED** [review]

**Review by lelia — COMMENTED** [review]
+1 to @pkalapat's suggestions - otherwise LGTM

File: components/Layout.tsx (hunk 5 above)
Comment (edwarnicke) [reply components/Layout.tsx]: I believe this is fixed in 6ed1a5c287472f9917ff1c9dc345ecaab15636d1

File: components/Layout.tsx (hunk 6 above)
Comment (edwarnicke) [reply components/Layout.tsx]: I believe this is fixed in 6ed1a5c287472f9917ff1c9dc345ecaab15636d1

Comment (pkalapat) [reply components/Layout.tsx]: LGTM.

**Comment by pkalapat** [comment]
The changes looks good to me.

**Review by detiber — APPROVED** [review]

**Pull request merged**